content/intro-to-storybook/angular/ja/conclusion.md

```diff
@@ -5,10 +5,10 @@ description: '今までの知識をまとめて、Storybook のテクニック
 
 Storybook で最初の UI を作成しましたね。お疲れ様でした！ここまでの章で UI コンポーネントを作成し、複合させ、テストし、デプロイする方法を学びました。同じように進めていれば、リポジトリーとデプロイされた Storybook は以下のリンクと同じようになっていることでしょう。
 
-[📕 **GitHub のリポジトリー: chromaui/learnstorybook-code**](https://github.com/chromaui/learnstorybook-code/tree/angular)
+[📕 **GitHub のリポジトリー: chromaui/learnstorybook-code**](https://github.com/chromaui/learnstorybook-code)
 <br/>
 
-[🌎 **デプロイされた Storybook**](https://angular--5ccbe484c994280020b6d128.chromatic.com/)
+[🌎 **デプロイされた Storybook**](https://master--5ccbe484c994280020b6d128.chromatic.com)
```

Comment:
@nagashimam Same here, we have an Angular deployment in Chromatic, as this should be "https://angular--5ccbe484c994280020b6d128.chromatic.com/"